### PR TITLE
Update MySQL (Old) jdbc driver

### PR DESCRIPTION
**Pull request by m1ngyuan**
Update mysql jdbc driver

### CODE REVIEW

**Comment by LonwoLonwo** [comment]
Hello @m1ngyuan 

Thanks for your contribution.
We will look into it.

File: plugins/org.jkiss.dbeaver.ext.generic/plugin.xml [hunk 1]
```diff
@@ -1423,7 +1423,7 @@
                         webURL="https://www.mysql.com/products/connector/"
                         supportedConfigurationTypes="MANUAL,URL"
                         description="Driver for MySQL (4.x and earlier)">
-                    <file type="jar" path="maven:/mysql:mysql-connector-java:RELEASE"/>
+                    <file type="jar" path="maven:/com.mysql:mysql-connector-j:RELEASE"/>
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.ext.generic/plugin.xml]:
```suggestion
                    <file type="jar" path="maven:/com.mysql:mysql-connector-j:RELEASE[8.2.0]"/>
```

Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.ext.generic/plugin.xml]: Please add a specific version. We can not be sure that the next release will not have vulnerabilities.

**Review by LonwoLonwo — APPROVED** [review]

**Comment by serge-rider** [comment]
https://github.com/dbeaver/dbeaver/issues/21906
Now I am not sure that we shoulod update legacy driver. It may break backward compatibility

**Comment by LonwoLonwo** [comment]
Because of this ticket #21906, we are not sure about backward compatibility.

My suggestion here is not to change the link for the latest but to use a specific old MySQL driver version:

maven:/mysql:mysql-connector-java:RELEASE[5.1.44]

**Comment by m1ngyuan** [comment]
> Because of this ticket #21906, we are not sure about backward compatibility.
> 
> My suggestion here is not to change the link for the latest but to use a specific old MySQL driver version:
> 
> maven:/mysql:mysql-connector-java:RELEASE[5.1.44]


Thank you for your advice. I believe you are right. 
Here are the relevant release notes. https://downloads.mysql.com/docs/connector-j-5.1-relnotes-en.pdf 
and I think we should upgrade to 5.1.49, which is the latest GA version that fixes some bugs in the MySQL 5. x branch

**Review by LonwoLonwo — APPROVED** [review]

**Comment by DariaMarkaryan** [comment]
Verified

**Pull request merged**